### PR TITLE
Identity live pipeline fixes

### PR DESCRIPTION
**Pull request by billwert**
Fixing a couple blocking bugs in our live pipeline

### CODE REVIEW

File: sdk/identity/azure-identity/pom.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.27.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+      <version>1.27.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-core-test;dependency} -->
```
Comment (alzimmermsft) [inline sdk/identity/azure-identity/pom.xml:67]: You may need unreleased_com.azure:azure-core as well

Comment (billwert) [reply sdk/identity/azure-identity/pom.xml:67]: I did, fixed.

**Review by alzimmermsft — APPROVED** [review]

**Pull request merged**